examples/HotPotQA/workflow.py

```diff
@@ -4,7 +4,7 @@
 
 import dspy
 
-gpt4o_mini = dspy.LM('gpt-4o-mini', max_tokens=1000)
+gpt4o_mini = dspy.LM('gpt-4o', max_tokens=1000)
```

Comment:
why gpt4o_mini has gpt-4o in the dspy definition?

Reply:
Whoops. Fixed.